src/App.js

```diff
@@ -1,55 +1,129 @@
 import React, { Component } from 'react';
 import Table from './components/FoodBanks.js';
-import data from './components/FoodBanks.json';
 import Button from '@material-ui/core/Button';
 import NavBar from './components/NavBar.js';
 import StartingPoint from './components/StartingPoint.js';
 import { geolocated } from "react-geolocated";
 import './App.css';
+import ReactMapboxGl, { Marker, ZoomControl} from 'react-mapbox-gl';
+import geojson from 'geojson';
+import geoFoodBanks from './components/geoBanks.json';
+import geoShelters from './components/geoShelters.json';
+import geoHealths from './components/geoHealth.json';
 
 
 class App extends Component {
-  
   render(){
-    const foodBanks = data
     const { isGeolocationAvailable, isGeolocationEnabled, coords } = this.props;
+    const Map = ReactMapboxGl({
+      accessToken:
+      'pk.eyJ1IjoiY29saW5waGlsbGlwczY3IiwiYSI6ImNrMGZvd2MyaDAxdG8zbHJ6MmFnNnZpaTUifQ.tJ63Dty8f2DWdS1jenXPCA'
+    },
+    
+    );
+    const geoFood = geojson.parse(geoFoodBanks, {Point:['lat','lng'], include:['Name']});
+    const geoShelter = geojson.parse(geoShelters, {Point:['lat','lng'], include:['Name']});
+    const geoHealth = geojson.parse(geoHealths, {Point:['lat','lng'], include:['Name']});
+
     return !isGeolocationAvailable ? (
       <div>Your browser does not support Geolocation</div>
   ) : !isGeolocationEnabled ? (
       <div>Geolocation is not enabled</div>
   ) : coords ? (
        <div className="App">
          
-        <h1>Social Services WebApp</h1>
+        <h1 className="title">Social Services WebApp</h1>
         <NavBar />
-        <Table foodBankData={foodBanks}/>
+        
+        
         <br/>
         <Button variant="contained" color="primary">
           Nearest Food Bank
         </Button>
+
+        <div className="map">
+{/* Map Function showing markers for all locations in json files. */}
+
+          <Map className="mapbox"
+            style={'mapbox://styles/mapbox/streets-v11'}      
+            center={[coords.longitude, coords.latitude]}
+            // for testing when outside of KC using line below for center
+            // center = {[-94.6, 39.025]}
+            containerStyle={{
+            height: '80vh',
+            width: '90vw'
+          }}
+          >            
+            
+          <ZoomControl/>
+          
+          <Marker
+          coordinates={[-94.6359718, 38.919853200000006]}
```

Comment:
Why is this hard coded?

Reply:
Good catch I updated the code and I think it has been updated to below

```suggestion
         coordinates={[coords.longitude, coords.latitude]}
```